ivy/functional/frontends/torch/blas_and_lapack_ops.py

```diff
@@ -210,3 +210,10 @@ def trapezoid(y, x=None, *, dx=None, dim=-1):
     if x is not None:
         y, x = torch_frontend.promote_types_of_torch_inputs(y, x)
     return ivy.trapz(y, x=x, dx=dx, axis=dim)
+
+
+@to_ivy_arrays_and_back
+def cumulative_trapezoid(y, x=None, *, dx=None, dim=-1):
+    if x is not None:
+        y, x = torch_frontend.promote_types_of_torch_inputs(y, x)
+    return ivy.cumulative_trapezoid(y, x=x, dx=dx, axis=dim)
```

Comment:
there is no cumulative_trapezoid function in ivy for you to use directly like this, please consider compositional approach or something else.

Reply:
Hello @Daniel4078 so should I make another PR for cumulative_trapezoid or continue the function development here only, by using the composition function?

Reply:
you should just continue here first as there might be some functions in the Ivy api that provide similar feature like cumulative_trapezoid but named differently.